src/gluonts/ev/aggregations.py

```diff
@@ -0,0 +1,95 @@
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
+from dataclasses import dataclass
+from typing import List, Optional, Union
+import numpy as np
+
+from gluonts.ev.helpers import axis_is_zero_or_none
+
+
+@dataclass
+class Aggregation:
+    def step(self) -> None:
+        raise NotImplementedError
+
+    def get(self) -> np.ndarray:
+        raise NotImplementedError
+
+    def reset(self) -> None:
+        raise NotImplementedError
```

Comment:
My feeling is we should remove this.

Reply:
It's not pretty but I'm not sure how to do without it.

Let's imagine the following use case: Constructing a `MetricEvaluator` and then calling it twice on different data without wanting to reconfigure things. This can be handy, I think.
But to do this, we need a way to reset things in between, right?

```
special_mse = MSE(forecast_type=0.7)(axis=1)  # only setup metric once

res_1 = special_mse.evaluate(experiment_1)
res_2 = special_mse.evaluate(experiment_2) # reset is called before internally
```